libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2127,6 +2128,57 @@ void GCS_MAVLINK::send_raw_imu()
 #endif
 }
 
+void GCS_MAVLINK::send_highres_imu()
+{
+    Vector3f accel;
+    Vector3f gyro;
+#if AP_INERTIALSENSOR_ENABLED
+    const AP_InertialSensor &ins = AP::ins();
+    accel = ins.get_accel(0);
+    gyro = ins.get_gyro(0);
```

Comment:
Don't you really want the primary IMU here - the one the vehicle is actually using?  Note that this can change over time and lead to discontinuities.